### PR TITLE
Added support for Apple watchOS and tvOS

### PR DESCRIPTION
**Pull request by justinboswell**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: builder/core/data.py [hunk 1]
```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
...
+        'architectures': {
+            'armv8': {
+                'variables': {
+                    'osx_architectures': 'arm64'
+                }
+            }
+        },
+        'variables': {
+            'exe': '',
+            'osx_deployment_target': '13.0',
```
Comment (kneekey23) [inline builder/core/data.py:358]: dont think this is correct. the latest version of tvOS is 7.0 and we should proably target two before so 5.0. 13 is the iOS version I believe.

File: builder/core/data.py [hunk 2]
```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
...
```
Comment (kneekey23) [inline builder/core/data.py:379]: same here. think this should be 5.0

File: builder/core/data.py [hunk 3]
```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
...
+            'osx_architectures': 'arm64;x86_64'
```
Comment (kneekey23) [inline builder/core/data.py:359]: can the architecture ever be x86_64?

File: builder/core/data.py [hunk 4]
```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
...
```
Comment (kneekey23) [inline builder/core/data.py:380]: same question here as above?

File: builder/core/data.py (hunk 1 above)
Comment (justinboswell) [reply builder/core/data.py:358]: good catch

**Review by kneekey23 — APPROVED** [review]
fix n ship and 2 questions

File: builder/core/data.py (hunk 2 above)
Comment (justinboswell) [reply builder/core/data.py:379]: tvOS tracks iOS, this one is still right :)

**Pull request merged**